### PR TITLE
Subaru Forester 2019

### PR DESCRIPTION
**Pull request by martinl**
This PR adds support for Subaru Forester 2019 (Global Platform). Tuning values and testing by Discord user dgranger15

Route ID for replay tests: `c321c6b697c5a5ff|2020-06-23--11-04-33`

Highway drive link: https://my.comma.ai/cabana/?route=c321c6b697c5a5ff%7C2020-06-23--11-04-33&exp=1624469159&sig=haDYm%2FnNboH2X%2BhlY4XFcbUVGXqykYgPAAlz3x9oSHk%3D&max=17&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Fc321c6b697c5a5ff%2Fff13c873e05ba2f5137fa23ad4126f7a_2020-06-23--11-04-33

### CODE REVIEW

File: selfdrive/test/test_car_models.py [hunk 1]
```diff
@@ -419,6 +419,7 @@ def get_route_log(route_name):
   TOYOTA.CHRH,
   TOYOTA.HIGHLANDERH,
   TOYOTA.HIGHLANDERH_TSS2,
+  SUBARU.FORESTER_2019,
```
Comment (adeebshihadeh) [inline selfdrive/test/test_car_models.py:422]: Can you add a test route?

File: selfdrive/car/subaru/interface.py [hunk 2]
```diff
@@ -38,6 +38,16 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0., 20.], [0., 20.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.2, 0.3], [0.02, 0.03]]
 
+    if candidate in [CAR.FORESTER_2019]:
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/interface.py:41]: no need to make this a list

Comment (martinl) [reply selfdrive/car/subaru/interface.py]: changed to single value

File: selfdrive/test/test_car_models.py (hunk 1 above)
Comment (martinl) [reply selfdrive/test/test_car_models.py]: Done

File: selfdrive/car/subaru/values.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 class CAR:
   IMPREZA = "SUBARU IMPREZA LIMITED 2019"
+  FORESTER_2019 = "SUBARU FORESTER 2019"
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/values.py:9]: The year is unnecessary here

**Comment by adeebshihadeh** [comment]
Uploaded the route to the CI bucket, tests should pass now.

File: selfdrive/car/subaru/values.py (hunk 3 above)
Comment (martinl) [reply selfdrive/car/subaru/values.py:9]: There is a preglobal PR (https://github.com/commaai/openpilot/pull/1489) which is already using FORESTER

Comment (adeebshihadeh) [reply selfdrive/car/subaru/values.py:9]: I think global should have priority given that it's the newest platform.

Comment (martinl) [reply selfdrive/car/subaru/values.py]: I removed the model year

File: selfdrive/car/subaru/values.py [hunk 4]
```diff
@@ -15,10 +16,19 @@ class CAR:
   {
     2: 8, 64: 8, 65: 8, 72: 8, 73: 8, 256: 8, 280: 8, 281: 8, 290: 8, 312: 8, 313: 8, 314: 8, 315: 8, 316: 8, 326: 8, 372: 8, 544: 8, 545: 8, 546: 8, 554: 8, 557: 8, 576: 8, 577: 8, 722: 8, 801: 8, 802: 8, 805: 8, 808: 8, 811: 8, 826: 8, 837: 8, 838: 8, 839: 8, 842: 8, 912: 8, 915: 8, 940: 8, 1614: 8, 1617: 8, 1632: 8, 1650: 8, 1657: 8, 1658: 8, 1677: 8, 1697: 8, 1759: 8, 1786: 5, 1787: 5, 1788: 8
   }],
+  CAR.FORESTER: [{
+  # Forester Sport 2019 (sirged)
```
Comment (adeebshihadeh) [inline selfdrive/car/subaru/values.py:20]: remove the usernames

Comment (martinl) [reply selfdrive/car/subaru/values.py]: Done

**Pull request merged**